engines/kingdom/constants.cpp

```diff
@@ -0,0 +1,5484 @@
+/* ScummVM - Graphic Adventure Engine
+ *
+ * ScummVM is the legal property of its developers, whose names
+ * are too numerous to list here. Please refer to the COPYRIGHT
+ * file distributed with this source distribution.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+ *
+ */
+
+#include "kingdom/kingdom.h"
+
+namespace Kingdom {
+
+const char *_rezNames[] = {
```

Comment:
There's enough constants that you should consider setting up a data file for them, though this could wait until post-merge

Reply:
I'm currently reluctant to switch to a DAT file, as I don't manage to work on an engine after that. This engine only support one of the ports of the game, I guess it could be interesting to have a look at at least the MAC port which has a different interface and uses Quicktime (including to display the fix image between two videos). So, yes, I totally agree it would be a lot cleaner... but right now I prefer to delay a bit, just in case.